Remove TD Jungle tileset.

pchote opened the pull request:
This tileset was community content, and does not have remastered assets. There is no realistic prospect to creating equivalent remastered assets, which leaves removal in order to achieve parity between classic and remastered modes.

JovialFeline commented:
There's at least one mismatch between the two tilesets on funpark01. Importing the terrain again might not hurt.
![Mismatch](https://github.com/user-attachments/assets/403cb639-70ef-4c07-b8b5-57d3dfcb5f2b)

PunkPun reviewed (commented):
Couldn't we just render non remastered assets for jungle tileset? Same way you'd render other assets from in community made maps?

pchote commented:
Technically yes, but I consider that to be a poor solution. We should be aiming for the remastered assets to become the default experience that is polished - theres no real prospect for a full remastered jungle tileset to be created, but I can see any smaller scale map assets being remade to provide remastered versions.

PunkPun commented:
I assume the proper solution would be to allow maps to provide their own tilesets? And then boot the jungle tileset?

pchote commented:
I'm not sure how map tilesets relate?

pchote committed:
Remove TD Jungle tileset.

This tileset was community content, and does not have remastered assets.
There is no realistic prospect to creating equivalent remastered assets,
which leaves removal in order to achieve parity between classic and
remastered modes.

pchote commented:
Updated with reimported funpark01 terrain.

Before and after, confirming that there are no substantial map changes that might impact gameplay:
<img alt="funpark-bleed" src="https://github.com/user-attachments/assets/3c260a53-d58d-49b8-abb7-358a710ab342" />
<img alt="funpark-pr" src="https://github.com/user-attachments/assets/78c655f3-166f-4802-bf14-4a979d8b657c" />

abcdefg30 commented:
Sad :(

PunkPun commented:
Having a mix of pixel art and HD art doesn't seem inherently bad. In Red Alert maps have tons of custom art, so a clash is inevitable.

Looking at the screenshots on discord of this mix, I actually think HD assets look great on the pixel art jungle tileset, I believe it even looks better than the remastered temperate tileset.

I'd say let's leave the decision of whether to remove jungle for later, closer to next release

pchote commented:
To be 100% honest, I am disappointed by the response to this PR (here and on Discord) and this has put a rather large damper on my recent reinterest in OpenRA.

My vision was for the remastered assets to become the preferred/default way to play OpenRA:TD, without compromising performance or polish. This necessarily means reevaluating and in some cases undoing previous choices to use community assets.

I'm not interested in arguing an uphill battle across multiple PRs, so if others don't buy into my vision then thats completely fine and I won't push it further. However, if this is the case, somebody else will have to pick up the project if they want to continue along a different path.

penev92 commented:
I feel OpenRA has (recently) developed a very unhealthy habit of being super unwelcoming. We should reconsider this policy as it is not conducive to progress.

abcdefg30 reviewed (approved):
Again, sad to see the tileset go, but we should make progress towards the remastered assets. It is not like it is lost forever.

penev92 commented:
What @abcdefg30 said.

penev92 commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/969932d316363f00af03202c353c970dca013c47...f455330dfbae4f0f4eeb8687eeef7389464fa003)
